apps/dsb-client-gateway-api/src/app/modules/message/service/dsb-message-pooling.service.ts

```diff
@@ -22,7 +32,10 @@ enum SCHEDULER_HANDLERS {
 @Injectable()
 export class DsbMessagePoolingService implements OnModuleInit {
   private readonly logger = new Logger(DsbMessagePoolingService.name);
-  private readonly websocketMode = this.configService.get('WEBSOCKET', WebSocketImplementation.NONE);
+  private readonly websocketMode = this.configService.get(
+    'WEBSOCKET',
+    WebSocketImplementation.NONE
+  );
   constructor(
```

Comment:
new line above

Reply:
This stuff I will be fixing with prettier this week, thanks